### PR TITLE
Add documentation to install Streams using an external database instead of the one provided by the Helm chart

### PR DESCRIPTION
**Pull request by benjaminranc**
Thank you for your contribution to the Axway Streams documentation.

## Describe the changes

Add documentation to install Streams using an external database instead of the one provided by the Helm chart.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links

### CODE REVIEW

File: content/en/docs/install/_index.md [hunk 1]
```diff
@@ -80,7 +80,90 @@ imagePullSecrets:
...
+    user: "my-streams-db-user"
+  rootUsername: "my-streams-db-root-user"
+```
+
+{{< alert title="Note" >}} It's recommended to have one user with privileges to create the database schema and another to manage the entries in the tables. If you want to use only one account, set the same user for `externalDatabase.db.user` and `externalDatabase.rootUsername`. {{< /alert >}}
+
+##### External database passwords
+
+Passwords are required for Streams microservices to securely connect to Mariadb.
+If you choose to use only one account, set the same password twice.
```
Comment (jfmarquet) [inline content/en/docs/install/_index.md:129]: I would remove this sentence.

File: content/en/docs/install/_index.md [hunk 2]
```diff
@@ -80,7 +80,90 @@ imagePullSecrets:
 
 ### MariaDB settings
 
-#### Password
+The database is automatically installed by default on your K8s cluster with the Streams Helm chart. But you can also use an external one if you preferred.
```
[the remark this answers is not in the document]
Comment (jfmarquet) [reply content/en/docs/install/_index.md:83]: typos: 
- cluster with Streams Helm chart. => remove the
- if you prefer

[the remark this answers is not in the document]
Comment (jfmarquet) [reply content/en/docs/install/_index.md:83]: Change second sentence and add in the doc that the recommended conf for production is with an external database.

**Review by allandenis38 — APPROVED** [review]

**Comment by allandenis38** [comment]
@benjaminranc @jfmarquet Could you please resolve conflicts ?

**Comment by benjaminranc** [comment]
> @benjaminranc @jfmarquet Could you please resolve conflicts ?

Done

**Review by lbadenhop — APPROVED** [review]

**Review by allandenis38 — APPROVED** [review]

File: content/en/docs/install/_index.md [hunk 3]
```diff
@@ -80,7 +80,90 @@ imagePullSecrets:
...
+
+To disable the Database (MariaDB) installation, you can either:
+
+* Edit the `values.yaml` file and set the `mariadb.enabled` entry as follow:
+
+```yaml
+mariadb:
+  enabled: false
```
[the remark this answers is not in the document]
Comment (allandenis38) [reply content/en/docs/install/_index.md:91]: Seems to be an issue related syntax highlight. To be investigated.

**Pull request merged**